lib/auth/clt.go

```diff
@@ -835,6 +835,11 @@ func (c *Client) UpsertUser(user types.User) error {
 	return trace.Wrap(err)
 }
 
+// UpdateAndSwapUser not implemented: can only be called locally.
```

Comment:
Manual backport, git got confused where to place this func. Other files were clean.